conf/test.config

```diff
@@ -23,22 +23,17 @@ params {
     // Input test data
     input   = "mMelMel3"
     project = ".sandbox"
+
+    // goat_taxon_search
+    taxon = 'Meles meles'
+    taxa_file = ''
+
+    // BUSCO
+    busco_lineages_path = './'
+
 }
 
 // Singularity settings - normally provided through `conf/sanger.config`
```

Comment:
Singularity settings should be provided by `-profile singularity` and `-profile sanger`, no need for them to be here.